docker-app/qfieldcloud/core/models.py

```diff
@@ -943,6 +943,10 @@ class StatusCode(models.TextChoices):
         )
         TOO_MANY_COLLABORATORS = "too_many_collaborators", _("Too many collaborators")
 
+    class PackagingOffliner(models.TextChoices):
+        QGISCORE = "qgiscore", _("QGIS Core Offline Editing")
+        PYTHONMINI = "pythonmini", _("Optimized Offliner")
```

Comment:
Feel free to suggest better human readable names (the ones passed to `_()`) function.

Reply:
QGIS Core Offline Editing sounds good, I wonder if we should already label it with "legacy" or (in case we wait for a few weeks before switching the default) we can add a "deprecated" to the name whenever we switch the default.

Personally I like "packager" more than "offliner", so I would probably go for "Optimized Packager".

Reply:
```suggestion
        QGISCORE = "qgiscore", _("QGIS Core Offline Editing (deprecated)")
        PYTHONMINI = "pythonmini", _("Optimized Packager")
```